Tests/RockLib.HealthChecks.AspNetCore.ResponseWriter.Tests/ResponseWriterTests.cs

```diff
@@ -145,29 +145,31 @@ public static class ResponseWriterTests
             .BeFalse();
     }
 
-    private static IEnumerable<JToken> GetHealthCheckEntries(string jsonBody) => 
-        JObject.Parse(jsonBody)!
+    private static IEnumerable<JToken> GetHealthCheckEntries(string jsonBody)
```

Comment:
Was there a reason to change to not be an expression-bodied member?

Reply:
another analyzer suggestion: Code body does not conform to code style settings: use statement body. I'll put it back

Reply:
I think we have an inconsistency across our .editorconfig settings with this one. Personally I like expression-bodied members :)